packages/core/src/__tests__/factory.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('graph', () => {
     let defaultLoaderId = 0;
     let customLoaderId = 0;
 
-    const createInstance = factory<any, any, any, any, { id: number }>({
+    const createInstance = factory<any, any, any, any, { id: number }>(undefined, {
```

Comment:
Why do you have to pass `undefined` as first parameter?

Reply:
Because of a hack someone did in order to simplify tests. See that bunch on `any` ?